src/utils/api/sendJUP.ts

```diff
@@ -33,10 +40,19 @@ export interface IBroadcastTransactionPayload extends IAPICall {
 async function sendJUP(unsigned: IUnsignedTransaction) {
   let signedTx: ISignedTransaction;
   let isValid: boolean;
+  let sendQty: string;
+
+  // set the appropriate quantity based on transaction type
+  if (Object.keys(unsigned.attachment).includes("quantityQNT")) {
```

Comment:
just wanted to say that if quantityQNT needs to be refactored someday to another property name, this will be easy to miss. there might be a way we can get around that problem, with say a 'template' variable. aka instantiate an object with the proper keys (with TS checking it properly) and then we do a function/if statement to confirm that they key names match between the test object and the template object (for a transaction with an Asset to transfer)

i think this is a low priority thing and could be put as an issue or ignored entirely.